blocks/hero/hero.css

```diff
@@ -96,7 +96,7 @@
 
   body.article main .section.hero-container {
     position: relative;
-    width: 100vw;
+    width: 100%;
```

Comment:
IIRC, I've set this to take the whole view port width rather than 100%. The parent of hero-container section is main and it's width is already reduced so, 100% will only cover some part only.

<img width="1728" alt="image" src="https://github.com/hlxsites/accenture-newsroom/assets/1553152/05c58cde-f9af-4fc0-8f19-22149863a6e8">

BTW, I don't see any vertical scroll in firefox
<img width="1728" alt="image" src="https://github.com/hlxsites/accenture-newsroom/assets/1553152/38115692-0237-4939-8362-2100d1cab895">

Reply:
the problem is not vertical scroll, its the horizontal scroll in firefox as indicated in the ticket